docs/guides/create_BYOK_groups.md

```diff
@@ -0,0 +1,94 @@
+## This examples will illustrate on how to create Cloud data control groups. We create AWS, Azure and GCP groups.
+
+* AWS
+
+```
+// Create AWS group
+resource "dsm_group" "aws_byok" {
+    name = "aws_byok"
+    description = "aws_byok"
+    hmg = var.aws_data
+}
+
+// aws data to create a group inside dsm
+variable "aws_data" {
+  type        = any
+  description = "The policy document. This is a JSON formatted string."
+  default     = <<-EOF
+    {
+    "url": "kms.<region-names>.amazonaws.com",
+    "tls": {
+      "mode": "required",
+      "validate_hostname": false,
+      "ca": {
+        "ca_set": "global_roots"
+      }
+    },
+    "kind": "AWSKMS",
+    "access_key": "<aws_access_key>",
+    "secret_key": "<aws_secret_key>",
+    "region": "<aws_region>",
+    "service": "kms"
+    }
+  EOF
+}
+```
+* Azure
+
+```
+// Create Azure group
+resource "dsm_group" "azure_byok" {
+  name = "azure_byok"
+  description = "azure_byok"
+  hmg = var.azure_data
+}
+
+// azure data to create a group inside dsm
+variable "azure_data" {
+  type        = any
+  description = "The policy document. This is a JSON formatted string."
+  default     = <<-EOF
+    {
+      "url": "<key_vault_url>",
+      "tls": {
+        "mode": "required",
+        "validate_hostname": false,
+        "ca": {
+          "ca_set": "global_roots"
+        }
+      },
+      "kind": "AZUREKEYVAULT",
+      "secret_key": "<aws_secret_key>",
```

Comment:
I guess this is fine, is it expected to provide sensitive information like this?